ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/FilterByDocType.java

```diff
@@ -100,8 +105,8 @@ public void processElement(@Element PubsubMessage message, OutputReceiver<Pubsub
         final String version = message.getAttribute(Attribute.USER_AGENT_VERSION);
         try {
           if (version == null || minVersion > Integer.parseInt(version)
-              || (limitLegacyDesktopVersion && isLegacyDesktop
-                  && 116 <= Integer.parseInt(version))) {
+              || maxVersion < Integer.parseInt(version) || (limitLegacyDesktopVersion
+                  && isLegacyDesktop && 116 <= Integer.parseInt(version))) {
             PerDocTypeCounter.inc(message.getAttributeMap(), "version_filter_rejected");
```

Comment:
I'm not sure if this should still inc the counter.

Reply:
I think it's fine. As far as I can tell this metric has been 0 for a while now, so we would get some visibility into how many messages we're dropping without it necessarily getting mixed up with messages dropped due to minVersion.